kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/exception/BigQueryConnectException.java

```diff
@@ -53,8 +53,9 @@ private static String formatInsertAllErrors(Map<Long, List<BigQueryError>> error
     for (Map.Entry<Long, List<BigQueryError>> errorsEntry : errorsMap.entrySet()) {
       for (BigQueryError error : errorsEntry.getValue()) {
         messageBuilder.append(String.format(
-            "%n\t[row index %d]: %s: %s",
+            "%n\t[row index %d]: %s: %s: %s",
```

Comment:
Based on the "if present" clause in the [javadoc](https://github.com/googleapis/java-bigquery/blob/3e3b1da0f9e4006a83ed824f8ed67a9798219bee/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryError.java#L82) for `BigQueryError::getLocation`, it seems possible this may return `null`, or some other value that may not be self-describing (I think I've seen something like `GLOBAL` for the location in the past).

Now that we're including a fair amount of information in here (location, reason, and message), do you think it may make sense to explicitly label the various components so that the error message looks something like `[row index <index>] (location <location>, reason <reason>): <message>`?

That way, if the location (or even the reason) is `null`, the result is still fairly human-readable:
```
[row index 12] (location null, reason null): Some error occurred
```

instead of:
```
[row index 12]: null: null: Some error occurred
```

Reply:
Yes, that makes sense to me! Thanks!